ddtrace/context.py

```diff
@@ -107,7 +107,7 @@ def dd_origin(self, value):
                 return
             self._meta[ORIGIN_KEY] = value
 
-    @deprecated("Cloning contexts will no longer be required in 0.50", version="0.50")
+    @remove(message="Cloning contexts will no longer be required in 0.50", removal_version="1.0.0")
```

Comment:
This change removes this deprecation warning: https://github.com/DataDog/dd-trace-py/pull/3283